From: Youhei SASAKI <uwabami@gfd-dennou.org>
Date: Tue, 28 Feb 2012 20:44:20 +0900
Subject: Add Encoding.default_external
Forwarded: not-needed

Add Encoding.default_external = "UTF-8" due to sbuild/pbuilder locale
settings. This might be Debian specific.

Signed-off-by: Youhei SASAKI <uwabami@gfd-dennou.org>
---
 test/functional/basic.rb |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/test/functional/basic.rb b/test/functional/basic.rb
index 3053b54..37d52f0 100755
--- a/test/functional/basic.rb
+++ b/test/functional/basic.rb
@@ -1,4 +1,5 @@
 # encoding: utf-8
+Encoding.default_external = "UTF-8" if defined? Encoding
 require 'test/unit'
 require File.expand_path('../../lib/assert_warning', __FILE__)
 
